assemble/bin/accumulo-cluster

```diff
@@ -439,7 +439,7 @@ function control_services() {
         hosts="SSERVER_HOSTS_$group"
         for sserver in ${!hosts}; do
           if canRunOnHost "$sserver"; then
-            execute_command "$operation" "$sserver" sserver "$group" "-g" "$group"
+            execute_command "$operation" "$sserver" sserver "$group" "-o" "sserver.group=$group"
```

Comment:
@dlmarion why is `$group` being passed twice? is the first one, before the `-o` still used/needed? Does it serve a different function than the one in the value of the `-o` option?